requirements.txt

```diff
@@ -0,0 +1,8 @@
+pandas>=0.15
+seaborn>=0.7.0
+scipy>=0.16.1
```

Comment:
scipy 0.17.1 will likely revert back the behavior (hoping to submit a scipy PR for that soon), so I think we're better off explicitly specifying two-sided. I figured I'd do that in a separate PR.